OmsiHook/OmsiSoundPack.cs

```diff
@@ -15,7 +15,7 @@ public class OmsiSoundPack : OmsiObject
         }*/
         public string FileName
         {
-            get => Memory.ReadMemoryString(Memory.ReadMemory<int>(Address + 0x8));
+            get => Memory.ReadMemoryString(Address + 0x8);
             set => Memory.WriteMemory(Address + 0x8, value);
```

Comment:
If you've changed the getter have you checked that the setter still works?

Reply:
seems to work